packages/components/src/components/date-picker/date-picker-native.tsx

```diff
@@ -4,7 +4,25 @@ import { toMoment } from '@deriv/shared';
 import Icon from '../icon';
 import Text from '../text';
 
-const Native = ({
+interface NativePropsT {
+    value: string;
+    id: string;
+    name: string;
+    label: string;
+    placeholder: string;
+    max_date: moment.Moment;
+    min_date: moment.Moment;
+    display_format: string;
+    data_testid: string;
+    hint?: string;
+    error?: string;
+    disabled: boolean;
+    onSelect: (new_date: string) => void;
+    onBlur: (event: React.FocusEvent<HTMLInputElement>) => void;
+    onFocus: (event: React.FocusEvent<HTMLInputElement>) => void;
+}
+
+const Native: React.FC<NativePropsT> = ({
```

Comment:
```suggestion
export type TDatePickerNativeProps = React.HTMLAttributes<HTMLInputElement> & {
    value: string;
    label: string;
    placeholder: string;
    max_date: moment.Moment;
    min_date: moment.Moment;
    display_format: string;
    data_testid: string;
    hint?: string;
    onSelect: (new_date: string) => void;
    onBlur: (event: React.FocusEvent<HTMLInputElement>) => void;
    onFocus: (event: React.FocusEvent<HTMLInputElement>) => void;
}

const Native = ({
```
Let's use type instead of interface to maintain the common style we have in the project, and extend native input props just like our component's name suggests👍🏻
Also, as I explained here https://github.com/niloofar-deriv/deriv-app/pull/110/files#r1158694816, we have to avoid using React.FC since it's not supported in React 18.